### PR TITLE
Prepare for 2.0.4 release

### PR DESCRIPTION
**Pull request by igable**

### CODE REVIEW

**Comment by igable** [comment]
I've got the tarball built. And the chart-release commands ready to go. Once I've got a review on this I'll pull the trigger.

**Review by scardena — APPROVED** [review]
looks good. is there any docs on what are the chart-release commands, and other necessary commands are necessary for this release? That would be helpful so other can help on the release process for this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 2.0.4 (2021-12-07)
 
 * Added the ability to add additional settings to the configmap of the stardog configmap
```
Comment (pdmars) [inline CHANGELOG.md]: super small thing: this one is missing its corresponding issue

**Review by pdmars — APPROVED** [review]
looks good, just the one item in the release notes missing its issue

File: CHANGELOG.md (hunk 1 above)
Comment (igable) [reply CHANGELOG.md]: Fixed. Thanks for spotting that.

**Comment by igable** [comment]
@scardena internal release process docs are coming.

**Pull request merged**